StaticAnalyzer/tools/apkid

```diff
@@ -0,0 +1,11 @@
+#!/usr/local/opt/python/bin/python2.7
+
+# -*- coding: utf-8 -*-
+import re
+import sys
+
+from apkid import main
```

Comment:
The yara-python fork is not available in pip?
This is a bit problem. For the end users we need to reduce the config as much as possible.

We might need to look into ways to automatedly clone and install yara rules. And fallback to old implementation if the install fails/ user did not install yara-python.